client/http/client.go

```diff
@@ -51,11 +52,24 @@ type Client interface {
 	GetHotWriteRegions(context.Context) (*StoreHotPeersInfos, error)
 	GetRegionStatusByKeyRange(context.Context, []byte, []byte) (*RegionStats, error)
 	GetStores(context.Context) (*StoresInfo, error)
+	/* Rule-related interfaces */
```

Comment:
We have two groups of API for the placement rule, maybe we can reduce it later.

Reply:
Yes, after replacing the existing TiDB code, we can reduce the exposed interfaces here.